ios/MaasCore/Sources/MaasCore/Api/Utils/ApiKotlinFlowPublisher.swift

```diff
@@ -2,16 +2,16 @@ import Combine
 
 class ApiKotlinFlowPublisher<T: AnyObject>: Publisher {
     
-    typealias Output = T
-    typealias Failure = ApiError
+    public typealias Output = T
+    public typealias Failure = ApiError
     
     private let flow: CFlow<ApiResult<T>>
     
-    init(_ flow: CFlow<ApiResult<T>>) {
+    public init(_ flow: CFlow<ApiResult<T>>) {
```

Comment:
Not sure what it means in Swift for a class to be internal but its members public 🤷